src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingExtensionMethods.cs

```diff
@@ -0,0 +1,132 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.SqlServer.Management.Sdk.Sfc;
+using Microsoft.SqlServer.Management.SqlScriptPublish;
+using Microsoft.SqlTools.ServiceLayer.Scripting.Contracts;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.Scripting
+{
+    public static class ScriptingExtensionMethods
```

Comment:
Fixed.

---
In reply to: [111650593](https://github.com/Microsoft/sqltoolsservice/pull/316#discussion_r111650593) [](ancestors = 111650593)